.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
```

Comment:
Add a label to filter the requests more easily. Then subscribe to the label to keep an eye on how this template is working out https://dart-github-label-notifier.firebaseapp.com/  (internal link).

Reply:
I've created two new labels:

1. cp: review
2. cp: approved

This could solve the problem around contributors having to specify which release their cherry pick applies to.  I have a low concern for contributors abusing the cherry pick system as they do not utilize the current cp: labels often.  If this becomes an issue, we can pivot.

Reply:
> contributors abusing the cherry pick system as they do not utilize the current cp: labels

Random people don't have permissions to add labels, but they can create new issues.  I'm not worried about #flutter-hackers abusing this.